mne/label.py

```diff
@@ -747,6 +747,9 @@ def center_of_mass(self, subject=None, restrict_vertices=False,
         subject = _check_subject(self.subject, subject)
         if np.any(self.values < 0):
             raise ValueError('Cannot compute COM with negative values')
+        if np.all(self.values == 0):
+            raise RuntimeWarning('Label values are all 0, treating as a structural label\n')
```

Comment:
Argh. That seems like a bug but it could break people's code in the case that they assume it's zero and set one or a subset of points to something other than zero.

I'm okay with that as a bugfix (under the BUG section of `whats_new.rst`) so no deprecation cycle, but let's see if @agramfort agrees before moving forward.